addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20415,6 +20415,11 @@ msgctxt "#39013"
 msgid "Virtual filesystems"
 msgstr ""
 
+#: xbmc/addons/GUIDialogAddonInfo.cpp
```

Comment:
mind adding a contextual help string?
`#. Message of a dialog, asking if related data of an add-on should also be removed on uninstall`

And as the addon_data folder can contain more than just settings (like caches etc), we could probably rewrite the message to something like:
`Would you also like to remove all related data (e.g. settings) of this add-on?`
No strong feelings about this though - just a though. Thanks spiff.